### ISSUE TITLE
CSV Import/Export for Lokalize

### ISSUE THREAD

**Issue by bomas13** [Cockpit-HQ/Cockpit#231]
Previously it was possible by adding a config var named "importKeys: true" to activate CSV import and export for Lokalize. Digging through the code I was not able to find any hint that there is any of this still possible.

Is there a plan to re-add this, and if so: when?

**Comment by aheinze** [comment]
yes, an export/import for keys is on the todo list. Should be implemented within the next 2 months 🤞

**Comment by joh-dietz** [comment]
@aheinze apparently the new version has now an import/export functionality, but the multiline import does not work anymore how it used to.

i added multiline translations to the project and exported the keys. in the .csv they seem to be exported correctly as multiline:

![Image](https://github.com/user-attachments/assets/2fe989d7-1f1a-490f-aa8c-1e5f013a29b1)
<img width="251" alt="Image" src="https://github.com/user-attachments/assets/060f170a-be49-44dd-94b6-5ccc76fe62af" />

but if i import the same file via the import button, only the first line of the value is considered.
![Image](https://github.com/user-attachments/assets/cd50e93b-9352-43c2-b31e-4638c470b25c)

**Comment by aheinze** [comment]
@joh-dietz Could you please create a new issue? Thanks!
